packages/fxa-auth-server/lib/metrics/amplitude.js

```diff
@@ -49,6 +49,7 @@ const EMAIL_TYPES = {
   passwordReset: 'reset_password',
   passwordResetAccountRecovery: 'account_recovery',
   postChangePrimary: 'change_email',
+  postLinkedAccount: 'login',
```

Comment:
Sorry to be pedantic, I think `postAddLinkedAccount` might be better. Similar to our post secondary name.

Reply:
Not at all, I'm anticipating a `postRemoveLinkedAccount` email now though 🙂